connector_ebisumart/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 Quartile Limited
+{
+    "name": "Connector Ebisumart",
+    "version": "12.0.1.0.0",
+    "author": "Quartile Limited",
+    "website": "https://www.quartile.co",
+    "category": "Connector",
+    "license": "AGPL-3",
+    "depends": ["product","connector"],
```

Comment:
@AungKoKoLin1997 Just a question - is there no need to use `connector_ecommerce`?